Tests/Marketplace/marketplace_services.py

```diff
@@ -1130,19 +1141,23 @@ def assert_upload_bucket_version_matches_release_notes_version(self,
             f'current branch version: {latest_release_notes}\n' \
             'Please Merge from master and rebuild'
 
-    def prepare_release_notes(self, index_folder_path, build_number):
+    def prepare_release_notes(self, index_folder_path, build_number, pack_was_modified=False):
         """
         Handles the creation and update of the changelog.json files.
 
         Args:
             index_folder_path (str): Path to the unzipped index json.
             build_number (str): circleCI build number.
+            pack_was_modified (bool): whether the pack modified or not.
+
         Returns:
             bool: whether the operation succeeded.
             bool: whether running build has not updated pack release notes.
+            str: packs latest release notes
         """
         task_status = False
         not_updated_build = False
+        packs_latest_release_notes = Pack.PACK_INITIAL_VERSION
```

Comment:
```suggestion
        packs_latest_release_notes_version = Pack.PACK_INITIAL_VERSION
```
I think its better to add a version suffix as this var contains the version and not the actual rn.